src/interaction/interactiveTarget.js

```diff
@@ -8,41 +8,39 @@
  * @example
  *      function MyObject() {}
  *
- *      Object.assign(
- *          MyObject.prototype,
- *          PIXI.interaction.interactiveTarget
- *      );
+ *      PIXI.interaction.interactiveTarget.call(MyObject.prototype)
  */
-export default {
+export default function ()
```

Comment:
I'd keep this as an object instead of a function - as a function you are instantiating `getTrackedPointers` for every InteractiveTarget. Also, it is easier to deprecate things on InteractiveTarget if it is an object we can attach properties to.